### DISCUSSION TITLE
How to merge bars from a lower timeframe to a higher timeframe?

### DISCUSSION THREAD

**Discussion by tiger2014** [DaveSkender/Stock.Indicators#1192] [Help and support]
For example, obtaining 5m bars from 1m bars.

**Comment by tiger2014** [comment]
https://dotnet.stockindicators.dev/utilities/#content

**Reply by DaveSkender** [reply]
https://dotnet.stockindicators.dev/utilities/#resize-quote-history

```csharp
var fiveMinBars = oneMinBars.Aggregate(PeriodSize.FiveMinutes);
```

Side note:  we have an open issue for `PeriodSize.Week`

- #1184
